plugins/languages/clangd_extensions.nix

```diff
@@ -0,0 +1,189 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  helpers = import ../helpers.nix {inherit lib;};
+in
+  with lib; let
+    borderOpt = let
+      bordersTy =
+        types.enum ["double" "rounded" "single" "shadow" "none"];
+    in
+      helpers.defaultNullOpts.mkNullable (types.either bordersTy (types.listOf bordersTy))
+      ''"none"'' "";
+  in {
+    options.plugins.clangd_extensions = {
+      enable = mkEnableOption "clangd_extensions, plugin implementing clangd LSP extensions";
+
+      package = mkOption {
```

Comment:
There's also `mkPackageOption` in the helpers now

Reply:
Ah right, will edit